gen/vk_api.jl

```diff
@@ -2131,7 +2319,7 @@ function vkCmdSetDepthBias(commandBuffer, depthBiasConstantFactor, depthBiasClam
 end
 
 function vkCmdSetBlendConstants(commandBuffer, blendConstants, fun_ptr)
-    ccall(fun_ptr, Cvoid, (VkCommandBuffer, Ptr{Cfloat}), commandBuffer, blendConstants)
+    ccall(fun_ptr, Cvoid, (VkCommandBuffer, NTuple{4, Cfloat}), commandBuffer, blendConstants)
```

Comment:
This doesn't feel right to me. `const array` argument type should decay to a pointer type.

Reply:
But this might be a bug in the old generator.

Reply:
The specification hasn't changed, so it must be a difference in the generator. It should be fixed in the new generator, right? Or at least it will be a better place to fix it if needed. I suggest to accept this small bug for now and have it fixed with the new generator.

Reply:
This should be fixed in the new generator.